frontend/src/components/SamplesList.tsx

```diff
@@ -204,6 +208,7 @@ export const SamplesList: FunctionComponent<ISampleListProps> = ({
                     where: {
                       hasMetadataSampleMetadata_SOME: {
                         OR: sampleFilterWhereVariables(value),
+                        [sampleQueryParamFieldName]: sampleQueryParamValue,
```

Comment:
This fix works by adding an additional parameter for each search: the IGO Request ID for the current Request view or the CMO Patient ID for the current Patient view.